tensornetwork/network.py

```diff
@@ -1116,14 +1131,14 @@ def split_node_full_svd(self,
       # i + 1 to account for the new edge.
       right_node.add_edge(edge, i + 1)
       edge.update_axis(i + len(left_edges), node, i + 1, right_node)
-    self.connect(left_node[-1], singular_values_node[0])
-    self.connect(singular_values_node[1], right_node[0])
+    self.connect(left_node[-1], singular_values_node[0], name=left_edge_name)
+    self.connect(singular_values_node[1], right_node[0], name=right_edge_name)
     self.nodes_set.remove(node)
     return left_node, singular_values_node, right_node, trun_vals
 
   def remove_node(self, node: network_components.BaseNode
-                 ) -> Tuple[Dict[Text, network_components
-                                 .Edge], Dict[int, network_components.Edge]]:
+                 ) -> Tuple[Dict[Text, network_components.
```

Comment:
Can you bring the `.Edge` to the line above? (Yapf doesn't always make the best choices)

Reply:
no problem